mobile/native/src/calculations/mod.rs

```diff
@@ -15,6 +16,28 @@ pub fn calculate_quantity(opening_price: f64, margin: u64, leverage: f64) -> f64
     cfd::calculate_quantity(opening_price, margin, leverage)
 }
 
+pub fn calculate_pnl(
+    opening_price: f64,
+    closing_price: f64,
+    quantity: f64,
+    leverage: f64,
```

Comment:
side note: it is pretty scary that we use f64 everywhere and hope we will always make the order right 😢

Reply:
indeed, we definitely should do refactoring rounds where we introduce our own types that should preferably wrap Decimal and not float 😬